### DISCUSSION TITLE
Issue with the joyride floater.

### DISCUSSION THREAD

**Discussion by ng2695** [gilbarbara/react-joyride#930] [General]
The floater is placed on the wrong position.

Here, it supposed to indicate the white spotlight, but placing at the` top:0, left:0`

Can someone help me solving this, please?

**Comment by gilbarbara** [comment]
Hey @ng2695 
Without a reproducible example is nearly impossible to understand what's happening. Please create a minimal reproducible example in codesandbox.
Is this only happening in a single step or all of them?

Here are a few things you can try: 
- Set a min-height: 100vh in the body.
- Change the step placement

**Comment by ng2695** [comment]
Thanks for answering!

yeah, I've already included both above changes. 

The actual problem was the rendering order. The target element took a few seconds to render, so the floater wasn't able to find the target to place itself. It worked after I added a few seconds of delay.

**Comment by ng2695** [comment]
Resolved!
